types/node/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Node.js 10.11
+// Type definitions for Node.js 10.11.x
```

Comment:
All other changes were to remove `.x`, why adding it here? That should violate `dt-header`.